File: backend/src/main/java/com/team01/backend/domain/user/dto/UserProfileImageRequest.java
package com.team01.backend.domain.user.dto;

import lombok.*;

 // 프로필 사진만 따로 수정할 때 사용하는 DTO입니다.
 // 사진 업로드 버튼이 따로 있는 UI 구조에 맞춰 독립적으로 설계했습니다.

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class UserProfileImageRequest {
    private String profileImage; // 변경할 이미지의 URL 혹은 파일 경로
}

File: backend/src/main/java/com/team01/backend/domain/user/dto/UserProfileImageRequest.kt
package com.team01.backend.domain.user.dto

import jakarta.validation.constraints.Pattern

data class UserProfileImageRequest(
    @field:Pattern(
        regexp = "^/static/images/.*$",
        message = "프로필 이미지는 /static/images/ 경로로 시작해야 합니다."
    )
    val profileImage: String?
)
